editor/sidebar/post-status/index.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from 'i18n';
+import PanelBody from 'components/panel/body';
+import FormToggle from 'components/form-toggle';
+
+/**
+ * Internal Dependencies
+ */
+import './style.scss';
+function PostStatus() {
+	return (
+		<PanelBody toggle={ __( 'Status & Visibility' ) }>
+			<div>
+				<div className="editor-sidebar-post-status__row">
+					<span>{ __( 'Pending review' ) }</span>
+					<FormToggle />
```

Comment:
Good catch, with my docker setup, I was thinking I'm in debug mode by default, but it wasn't the case.